docs/sources/flow/reference/components/prometheus.scrape.md

```diff
@@ -54,38 +54,40 @@ Name | Type | Description | Default | Required
 `label_limit`              | `uint`     | More than this many labels post metric-relabeling causes the scrape to fail. | | no
 `label_name_length_limit`  | `uint`     | More than this label name length post metric-relabeling causes the scrape to fail. | | no
 `label_value_length_limit` | `uint`     | More than this label value length post metric-relabeling causes the scrape to fail. | | no
+`bearer_token`             | `secret`   | Bearer token to authenticate with. | | no
+`bearer_token_file`        | `string`   | File containing a bearer token to authenticate with. | | no
+`proxy_url`                | `string`   | HTTP proxy to proxy requests through. | | no
+`follow_redirects`         | `bool`     | Whether redirects returned by the server should be followed. | `true` | no
+`enable_http2`             | `bool`     | Whether HTTP2 is supported for requests. | `true` | no
+
+Only 1 of the following arguments or blocks can be provided
+- `bearer_token`
+- `bearer_token_file`
+- `basic_auth`
+- `authorization`
+- `oauth2`
```

Comment:
This feels like maybe the wrong place to put this. Should this be documented in the blocks section instead?